### ISSUE TITLE
Warning being shown when running the tool

### ISSUE THREAD

**Issue by macintacos** [aheckmann/m#81]
When I run `m`, I get some warning about using `egrep`:

```
$ m
egrep: warning: egrep is obsolescent; using ggrep -E
egrep: warning: egrep is obsolescent; using ggrep -E
```

I'm running macOS 12.6.1.

**Comment by stennie** [comment]
@macintacos Can you share some more details:
 - `egrep --version`
 - `which egrep`
 
I'm not getting the same warning with the default macOS `/usr/bin/egrep` and also don't have `ggrep` so I suspect you have a newer  version of GNU `grep` installed. That could still be handled more gracefully, I just need to make sure to test the expected versions.

Thanks,
Stennie

**Comment by stennie** [comment]
This is fixed by way of #87
